GVFS/GVFS/Program.cs

```diff
@@ -13,7 +13,11 @@ public class Program
         public static void Main(string[] args)
         {
             GVFSPlatformLoader.Initialize();
-            GVFSPlatform.Instance.KernelDriver.RegisterForOfflineIO();
+            if (!GVFSPlatform.Instance.KernelDriver.RegisterForOfflineIO())
+            {
+                Console.WriteLine("Unable to register for OfflineIO");

```

Comment:
What would you think of us providing a bit more detail here?  

Maybe something like:

> "Unable to register with the kernel for offline I/O.  Ensure that VFS for Git installed successfully and try again"